### PR TITLE
Wall Of Wisdom by Fox/Whitzy

### PR DESCRIPTION
**Pull request by Fox3058**

### CODE REVIEW

**Comment by AnyBlue** [comment]
Please rename the Screenshot on screenshot

File: wall_of_wisdom/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+team.1.pos = (-5,8,94)
+r = 115
+initial_stuff = Default:cobble 50,default:torch 20,default:pick_steel
```
Comment (Avyukt1) [inline wall_of_wisdom/map.conf:3]:
```suggestion
initial_stuff = default:cobble 50,default:torch 20,default:pick_steel
```
You shouldn't use capital letters else that will not give the correct item.

**Review by Avyukt1 — CHANGES_REQUESTED** [review]

**Comment by Avyukt1** [comment]
This map is **too plain**. add more forest or other constructions.

This map is just like The Wall by Thomas-S

**Comment by JostP** [comment]
re-open when you finish

**Pull request closed without merging**